MODEL-METADATA.md

```diff
@@ -33,3 +33,32 @@ which label will be chosen as the predicted label.
 ## Options specific to training models
 * trainOnProject (optional): A hash with the pid of a project you would like to train your new model or version 
 on. If this is supplied, the code you supplied will start to run against this pid automagically. 
+
+### Validation Schema
+The validation schema is used to define input and output requirements for the training model.  The validation is used to
```

Comment:
can you change out the terminology from training model -> custom task in this file?